int64 index
int64 id_a
int64 id_b
float64 value
